Domains/StructuralMaterials/StructuralMaterials.ecschema.xml

```diff
@@ -11,7 +11,6 @@
     </ECEntityClass>	
     <ECEntityClass typeName="GenericMechPropertiesAspect" description="Mechanical Properties for a Generic Material (i.e. other than Aluminum, Concrete or Steel)" displayLabel="Generic Structural Mechanical Properties" modifier="None">
         <BaseClass>MechPropertiesAspect</BaseClass>
-        <ECProperty propertyName="Density" typeName="double" description="Density" displayLabel="Density" kindOfQuantity="AECU:DENSITY" />			

```

Comment:
We've accidentally released out product with this schema. Would it be possible to mark this property as deprecated instead? And also update schema version to 1.0.1?

Reply:
@laurynas111 done - just pushed.  Both good suggestions.